starters/next-drupal-starter/pages/articles/[...slug].js

```diff
@@ -1,50 +1,23 @@
 import { NextSeo } from "next-seo";
-import { IMAGE_URL } from "../../lib/constants.js";
 import { isMultiLanguage } from "../../lib/isMultiLanguage";
 import { fetchJsonapiEndpoint } from "@pantheon-systems/drupal-kit";
 import {
   getCurrentLocaleStore,
   globalDrupalStateAuthStores,
 } from "../../lib/drupalStateContext";
 
-import Image from "next/image";
-import Link from "next/link";
+import Article from "../../components/article.js";
```

Comment:
Really excited about how clean these templates will look after this refactoring, and also some of the stuff we're planning on doing related to static paths and preview.